assets/src/edit-story/components/checklist/constants.js

```diff
@@ -80,6 +80,11 @@ export const ACCESSIBILITY_COPY = {
   },
   imagesMissingAltText: {
     title: __('Add assistive text to images', 'web-stories'),
+    // TODO: add `Learn more` link here to the footer?
+    footer: __(
+      'Optimize accessibility and indexability with meaningful text to better assist users.',
+      'web-stories'
+    ),
```

Comment:
This card was missing the footer as shown in the designs. Did we not add a footer here on purpose?
![image](https://user-images.githubusercontent.com/22185279/125873800-0a524b9f-1a82-4574-9d31-87881d2ae33d.png)

Reply:
nope, that was just missed. thank you!